File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for(let i=0; i<commands.length; i++){
        let t = commands[i]
        answer.push(array.slice(t[0]-1, t[1]).sort((a,b) => a-b).at(t[2]-1))
    }
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
# i번째 숫자부터 j번째 숫자까지 자르고 정렬 후 -> k번째 수
def solution(array, commands):
    answer = []
    
    for T in range(len(commands)):
        i,j,k = commands[T]
        lts = sorted(array[i-1:j])
        # print(lts)
        answer.append(lts[k-1])
    return answer